src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,151 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.exception.InvalidInputDataException;
+import core.basesyntax.service.exception.UserAlreadyExistsException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String VALID_PASSWORD = "validPassword123";
+    private static final Integer VALID_AGE = 33;
+    private static final Integer NEGATIVE_AGE = -1;
+    private static final Integer AGE_LESS_THAN_18 = 17;
+    private static final String SHORT_LOGIN = "usr";
+    private static final String SHORT_PASSWORD = "pwd";
+    private static final String EXISTING_LOGIN = "existingUser";
```

Comment:
Make sure to use meaningful and descriptive constant names that reflect their purpose within the tests. For example, 'VALID_LOGIN' could be 'VALID_USER_LOGIN' to clarify its usage.